client/extensions/woocommerce/app/settings/taxes/taxes-taxjar.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionHeader from 'woocommerce/components/action-header';
+import Card from 'components/card';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import ExternalLink from 'components/external-link';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import Main from 'components/main';
+import SettingsNavigation from '../navigation';
+
+class SettingsTaxesTaxJar extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		site: PropTypes.shape( {
+			options: PropTypes.shape( {
+				admin_url: PropTypes.string.isRequired,
+			} ),
+		} ),
+	};
+
+	onDeactivate = event => {
+		event.preventDefault();
+	};
+
+	render = () => {
+		const { className, site, translate } = this.props;
+
+		const breadcrumbs = [
+			<a href={ getLink( '/store/settings/:site/', site ) }>{ translate( 'Settings' ) }</a>,
+			<span>{ translate( 'Taxes' ) }</span>,
+		];
+
+		const adminUrl =
+			site.options.admin_url +
+			'admin.php?page=wc-settings&tab=integration&section=taxjar-integration';
+		const pluginUrl = getLink( '/plugins/taxjar-simplified-taxes-for-woocommerce/:site', site );
+
+		return (
+			<Main className={ classNames( 'settings-taxes', className ) }>
+				<ActionHeader breadcrumbs={ breadcrumbs } />
+				<SettingsNavigation activeSection="taxes" />
+				<div className="taxes__tax-jar">
+					<ExtendedHeader
+						label={ translate( 'Tax Plugin Detected' ) }
+						description={ translate( "A third-party plugin is managing your store's taxes." ) }
+					/>
+					<Card>
+						<p>
+							{ translate( "Your store's taxes are being managed by a third-party plugin. " ) }
+							<ExternalLink icon href={ adminUrl } rel="noopener noreferrer">
+								{ translate( 'Plugin settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Edit plugin settings' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).